docs/4.3/quickstart.md

```diff
@@ -274,6 +279,88 @@ $ echo "Awesome!"
 # check out your shared ssh session between two CLI windows
 ```
 
+## Run Teleport using Docker
+
+We provide pre-built Docker images for every version of Teleport. These images are hosted on quay.io.
+
+- [All tags under `quay.io/gravitational/teleport` are Teleport Community images](https://quay.io/repository/gravitational/teleport?tag=latest&tab=tags)
+- [All tags under `quay.io/gravitational/teleport-ent` are Teleport Enterprise images](https://quay.io/repository/gravitational/teleport-ent?tag=latest&tab=tags)
```

Comment:
Can you put this into the Enterprise Quickstart?

Reply:
Do you just want the quay.io link in the Enterprise quickstart, or the whole guide? Maybe I can link back to this page rather than duplicating the content?

Reply:
I would duplicate the content for now, I know this isn't ideal but we are trying to have less options and focus on community edition for the current quickstart.